settings.py

```diff
@@ -97,8 +97,17 @@
         'redirect_url': 'http://localhost:3000/api/v3/accounts/authenticate/',
         'scope': 'profile',
     },
+    'code-manager': {
+        'client_id': env('CODE_MANAGER_FXA_CLIENT_ID', default='CHANGE_ME'),
+        'client_secret': env('CODE_MANAGER_FXA_CLIENT_SECRET', default='CHANGE_ME'), # noqa
```

Comment:
can you use the dashboard to create some credentials (like the others do)

Reply:
yep